.github/workflows/test-on-push.yml

```diff
@@ -160,3 +160,22 @@ jobs:
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
           MSG_MINIMAL: ref
           SLACK_FOOTER: 'Job <https://github.com/softwaremill/fablo/commit/${{ github.sha }}/checks|*${{ github.workflow }}/${{ github.job }}*> ended with status: *${{ job.status }}*'
+
+  test-dco-sign-off:
+    needs: test-03-private-data
```

Comment:
daj to sprawdzenie może zaraz po tym, jak main przejdzie, nie ma co wydłużać całego procesu